assets/js/payment-methods-demo/payment-methods/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { paypalSvg } from './paypal';
+import { ccSvg } from './cc';
+
+export const paypalPaymentMethod = {
+	id: 'paypal',
+	label: <img src={ paypalSvg } alt="" />,
```

Comment:
Is the intention to always use the `label` property for icons or it's only in this demo content? I would rather rename it to `icon` if we are planning to always set the label to an image.

Reply:
This is just a quick mockup for a demo so I just used an image for now.  Currently the payment method will be given full leeway on how it prepares this integration.  It's most likely the payment method will return a component that displays an image icon but is actually a component with a onClick handler etc.  As a registered express payment method, it will receive props from the checkout that it can use for both processing the payment and also communicating state with the checkout. 

I'm not sure we'll stay with the `label` prop name but I think `icon` is too specific.  We could even change this to `component` maybe.

Reply:
Okay I wrote this thinking about the wrong thing.  This is not for the express payment method but is for the "regular" payment method so label here refers to the label used in the tab.  I think I'll leave this as is because payment methods might decide to return a text label as opposed to an image. 

The api for this will be documented and is somewhat similar to that of [composite-checkout](https://github.com/Automattic/wp-calypso/tree/master/packages/composite-checkout#payment-methods) which also uses the `label` property to represent a `React.node`.